dist/utils/index.js

```diff
@@ -17,18 +17,6 @@ var _path = require("path");
 
 var _path2 = _interopRequireDefault(_path);
 
-var _request = require("request");
```

Comment:
ignore these `dist` changes, these are auto generated when we run `npm run build` on our src files.. scroll down to index.js in src/utils for the juicy stuff